src/index.html

```diff
@@ -2,13 +2,63 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="menu">
+      <div class="menu__container">
+        <a href="#home" class="menu__logo">
+          <img src="./images/logo.png" alt="logo">
+        </a>
+
+        <nav class="menu__nav nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a class="nav__link is-active" href="#apple">apple</a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#samsung">samsung</a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#smartphones">smartphones</a>
+            </li>
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#computers"
+                data-qa="hover"
+              >laptops &amp; computers</a>
```

Comment:
It will be more clear to separate this line. And you can you ampersand symbol directly
```suggestion
              >
                laptops & computers
              </a>
```